Makefile

```diff
@@ -1,18 +1,20 @@
-.PHONY: test-unit test-e2e
+.PHONY: test-unit test-e2e pytest
```

Comment:
don't need this.  you only need a phony if you're trying to ensure there's no collision between a make target and a file.

pytest isn't a target

Reply:
that's an accident. will fix. I was going to make a `pytest` target but then thought better of it, exactly related to your other comment.